### ISSUE TITLE
Erase statement crashes on a method with ByRef parameter

### ISSUE THREAD

**Issue by Kr00l** [WaynePhillipsEA/twinbasic#367]
**Describe the bug**
There is a compilation error on the method EraseByteArray in below example.

**To Reproduce**
```
Public Sub Main()
    
    Dim B() As Byte
    ReDim B(0 To 3) As Byte
    
    Call EraseByteArray(B())
    
    MsgBox "Done"
    
End Sub

Private Sub EraseByteArray(ByRef B() As Byte)
    Erase B()
End Sub
```

**Expected behavior**
The code should work as in VBx.

**Screenshots**
![image](https://user-images.githubusercontent.com/33756666/130061658-98bf5f31-ce31-4e0a-b1c6-f625da92a902.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - twinBASIC compiler version [e.g. 0.10.4617]

**Additional context**
n/a

**Comment by WaynePhillipsEA** [comment]
Good catch, thanks.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.4755, thanks for reporting
